recipes/full_finetune_distributed.py

```diff
@@ -259,6 +267,10 @@ def _setup_model(
             ),
         )
 
+        if mode_tie:  # Tie the weights of the model if required
```

Comment:
```suggestion
        if model_tie:  # Tie the weights of the model if required
```

Reply:
Thank you for your suggestion, I have fixed it in the latest version.